assig2/src/presenter/filterQuestionsPresenter.js

```diff
@@ -0,0 +1,12 @@
+import model from "../model/model";
+
+class CreateQuestionPresenter {
```

Comment:
Wrong class name. Also, this presenter does not seem to do anything really (as when it is called, you are already on this route).